### PR TITLE
Pull-out_electrical_store for Voron 2.4

### PR DESCRIPTION
**Pull request by zzp8202**
* [ ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ ] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ ] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ ] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

### CODE REVIEW

File: printer_mods/ZZP8202/Pull-out_electrical_store/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Voron 2.4 Pull-out electrical Store 抽拉式电器仓
+
+## 适用
+
+| Creator | Mod title | Description | V0 | V1.8 | V2.4 | VSW | Trident |
```
Comment (cbc02009) [inline printer_mods/ZZP8202/Pull-out_electrical_store/README.md:5]: this doesn't need to go in your readme file. it's only for the main one in the printer_mods directory.

File: printer_mods/ZZP8202/Pull-out_electrical_store/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Voron 2.4 Pull-out electrical Store 抽拉式电器仓
+
+## 适用
+
+| Creator | Mod title | Description | V0 | V1.8 | V2.4 | VSW | Trident |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+|ZZP8202|[Pull-out_electrical_store](https://github.com/zzp8202/Pull-out_electrical_store)| Voron 2.4 Pull-out electrical Store |:x:|:x:|:heavy_check_mark:|:x:|:x:|
+
+Only aimed at Voron V2.4 350mm printer now.
```
Comment (cbc02009) [inline printer_mods/ZZP8202/Pull-out_electrical_store/README.md:9]: Can you put this in bold so users can't miss it?

File: printer_mods/ZZP8202/Pull-out_electrical_store/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+## Objectives
+
+The objectives for this project are:
+此项目的目标是：
+
+- 装逼* Just cool
+- 
+## Bill of Materials (BOM)
+
+待补充 to be supplemented
```
Comment (cbc02009) [inline printer_mods/ZZP8202/Pull-out_electrical_store/README.md:37]: We need a full BOM for the project.

File: printer_mods/ZZP8202/Pull-out_electrical_store/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+![图片](Photos/Voron_V2.4_Pull-out_electrical_store.PNG)
+### video
+https://www.bilibili.com/video/BV1iS4y1w7My?share_source=copy_web
+
+https://www.youtube.com/embed/usf8aM6o3Y0
+
+https://www.bilibili.com/video/BV12i4y1U7ux?share_source=copy_web
+
+https://www.youtube.com/embed/9XUpTigEtQ4
```
Comment (cbc02009) [inline printer_mods/ZZP8202/Pull-out_electrical_store/README.md:26]: Written instructions are also very helpful for users.

**Review by cbc02009 — CHANGES_REQUESTED** [review]

File: printer_mods/ZZP8202/Pull-out_electrical_store/README.md [hunk 5]
```diff
@@ -1,42 +1,69 @@
...
 
-待补充 to be supplemented
+>| **Category**|Part Name |**QTY**|Remark
+>| ----------------| :------------ | :--------------- | :--------------- |
+>|Printed Parts|Mini12864_Case|X1||
+>|Printed Parts|Front panel L|X1||
+>|Printed Parts|Front panel R|X1||
+>|Cables|24AWG Silicone Cable| X26M |Additional |
+>|Cables|20AWG Silicone Cable| X4M |Additional |
+>|Cables|[Cable Chain](https://www.amazon.com/Befenybay-Internal-Flexible-Machines-10mmX20mm/dp/B07SFFT1K5/ref=sr_1_3?crid=39NLBNYTC6OZX&keywords=Cable%2BChain&qid=1650390801&sprefix=cable%2Bchain%2Caps%2C975&sr=8-3&th=1)|X1|10*20(Inner H x Inner W)  (R18max)|
```
Comment (cbc02009) [inline printer_mods/ZZP8202/Pull-out_electrical_store/README.md:48]: everything except `https://www.amazon.com/Befenybay-Internal-Flexible-Machines-10mmX20mm/dp/B07SFFT1K5` can be deleted from this link. it's just tracking data.

File: printer_mods/ZZP8202/Pull-out_electrical_store/README.md [hunk 6]
```diff
@@ -1,42 +1,69 @@
...
+>|Hardware|STEEL CORNER|X8||
+>|Motion|[450mm Guide_rail](https://www.amazon.com/dp/B08C9PK2L8/ref=syn_sd_onsite_desktop_390?pd_rd_plhdr=t&spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUEzSVlYOU5IVk4xRENCJmVuY3J5cHRlZElkPUEwMjQzNTA3MlNCNkFTRUlVVlpGSiZlbmNyeXB0ZWRBZElkPUEwMzc2MDk2RDBQOEswSTI1VUdaJndpZGdldE5hbWU9c2Rfb25zaXRlX2Rlc2t0b3AmYWN0aW9uPWNsaWNrUmVkaXJlY3QmZG9Ob3RMb2dDbGljaz10cnVl&th=1)|X1Pair||
```
Comment (cbc02009) [inline printer_mods/ZZP8202/Pull-out_electrical_store/README.md:50]: same with this link delete /ref and everything after.

**Review by cbc02009 — CHANGES_REQUESTED** [review]

**Comment by cbc02009** [comment]
One last question, do you know if this will fit with the v2.4r2 z assemblies? If it won't or you're not sure you might want to add a note saying that it's only for v2.4r1.

**Comment by zzp8202** [comment]
> One last question, do you know if this will fit with the v2.4r2 z assemblies? If it won't or you're not sure you might want to add a note saying that it's only for v2.4r1.

Based on observations, it is compatible with V2.4 R2 except the side panels (option), anyway, I added the note in the readme.

BTW: Thank you for your hard work. Voron is a great project.

**Comment by cbc02009** [comment]
Sorry, just noticed there's a space in the [STEEL CORNER_LBSB6-2020_X8.pdf](https://github.com/VoronDesign/VoronUsers/pull/690/files#diff-be7a8f8e37cb0fb26622aa862bc4497467455be451238676aa295789b10c4c61) filename. please remove it (and make sure there aren't any more that I missed...) then we should be all set.

**Comment by zzp8202** [comment]
> Sorry, just noticed there's a space in the [STEEL CORNER_LBSB6-2020_X8.pdf](https://github.com/VoronDesign/VoronUsers/pull/690/files#diff-be7a8f8e37cb0fb26622aa862bc4497467455be451238676aa295789b10c4c61) filename. please remove it (and make sure there aren't any more that I missed...) then we should be all set.

It's done.

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**